base/libgit2/consts.jl

```diff
@@ -271,12 +274,14 @@ Option flags for `GitRepo`.
                         CREDTYPE_USERNAME           = Cuint(1 << 5),
                         CREDTYPE_SSH_MEMORY         = Cuint(1 << 6))
 
+if LibGit2.version() >= v"0.24.0"
```

Comment:
That's probably the most useful thing we'll get from the upgrade and I'm planning on making use of it at some point prior to 0.5 final, yes. Since Milan opened this, I'm going to guess Fedora Rawhide upgraded to 0.24.